src/dist/replication/lib/bulk_load/replica_bulk_loader.cpp

```diff
@@ -210,7 +210,8 @@ error_code replica_bulk_loader::do_bulk_load(const std::string &app_name,
     switch (meta_status) {
     case bulk_load_status::BLS_DOWNLOADING:
         // TODO(heyuchen): add restart downloading status check
```

Comment:
This comment can be removed now?

Reply:
This comment could not remove now, there will be other possible status in further pull request.